### ISSUE TITLE
No check for aria-describedby linking in text extractions/image descriptions

### ISSUE THREAD

**Issue by martinpub** [nlbdev/nordic-epub3-dtbook-migrator#519]
The 2020-1 guidelines [require text extractions/image descriptions to be put in a child aside element inside the figure, with proper linking between the img element and the aside using aria-describedby](https://format.mtm.se/nordic_epub/2020-1#text-extraction-from-images). However, there seems to be no check for aria-describedby linking in the validator.

The example from the guidelines would currently still be valid if aria-describedby is omitted, which would result in missing accessibility semantics and potentially rendering/skippability issues in reading systems.

A fix should introduce the following checks:

1. That aria-describedby is present on the img element of the figure
2. Link integrity from the aria-describedby reference to a matching fig-desc aside in the same figure container

**Comment by josteinaj** [comment]
I'm starting my vacation this weekend. I'll be working a bit during week 29 (July 18th - July 22nd), so if there's a PR by then, I can have a look at it and hopefully make a release. Otherwise I'll be back from vacation August 9th, so I can look more at it during August.

**Comment by martinpub** [comment]
Thanks @josteinaj! I think the time frame is OK, most people will be on vacation, just wanted to log this for now. Enjoy your vacation!

**Comment by josteinaj** [comment]
@martinpub @kalaspuffar solved?

**Comment by kalaspuffar** [comment]
To my knowledge, yes.
